helm/hpcc/templates/_helpers.tpl

```diff
@@ -825,6 +825,24 @@ A kludge to ensure until the mount of a PVC appears (this can happen with some t
       mountPath: {{ .volumePath | quote }}
 {{- end }}
 
+{{/*
+Inject container to perform any post plane initialization validation
+Pass in dict with volumeName, volumePath and cmds
+*/}}
+{{- define "hpcc.validatePlaneScript" -}}
+- name: {{ printf "validate-plane-script-container-%s" .volumeName }}
+  {{- include "hpcc.addImageAttrs" . | nindent 2 }}
+  command: ["/bin/bash", "-c"]
```

Comment:
minor: Slightly strange inconsistency between this and the wait for mount container - sh v bash and the way -c is passed.

Reply:
agree, but there's about 50% use cases of /bin/sh vs /bin/bash in various places in _helpers.tpl
It is the only place that has "-c" as part of the command list though, I'll move it to 1st arg to be more consistent.

bash is more rich, I think it's okay to use here - perhaps should unify on it elsewhere?